lib/hobbies.js

```diff
@@ -0,0 +1,76 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var cors = require('cors');
+var request = require('request');
+var $ = require('cheerio');
+
+
+function hobbiesRoute() {
```

Comment:
I don't think you need the function here. You can just create a router an export the router directly.